### PR TITLE
Added a new example with scrolling text

### PR DESCRIPTION
**Pull request by didaquis**
Added a new example with scrolling text. Fixed typo mistake. Clarify instructions for setting LCD Address.

### CODE REVIEW

**Comment by the-raspberry-pi-guy** [comment]
Hey! Thanks for the PR! I currently don't have the electronics to test these new additions - can you just confirm that the stuff you have added/changed doesn't cause my YouTube tutorial to no longer work? I'll merge after that!

File: lcddriver.py [hunk 1]
```diff
@@ -12,8 +12,13 @@
...
 from time import *
 
-# LCD Address
+# LCD Address 
+# Usually you will have to use one of the two provided values below.
+# If you prefer, you can check your LCD address with the command: "sudo i2cdetect -y 1"
+# This is a common LCD address.
 ADDRESS = 0x27
+# This is another common LCD address.
+#ADDRESS = 0x3f
```
Comment (didaquis) [inline lcddriver.py]: For some users `0x27` is the correct value but other people require change this value to `0x3f`. 
If you read the comments of your video tutorial, you will see that many people need to change that value.

**Comment by didaquis** [comment]
> can you just confirm that the stuff you have added/changed doesn't cause my YouTube tutorial to no longer work?

Yes, I confirm. I made right now some changes for guarantee that.

This PR introduce an isolated new example and a link to your video tutorial on README file

File: demo_lcd.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Demo program for the I2C 16x2 Display from Ryanteck.uk
 # Created by Matthew Timmons-Brown for The Raspberry Pi Guy YouTube channel
 
-# Import necessary libraries for commuunication and display use
+# Import necessary libraries for communication and display use
```
Comment (didaquis) [inline demo_lcd.py]: Fixed typo!

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # LCD
-This repository contains all of the code for interfacing with a 16x2 Character I2C LCD Display. This accompanies my YouTube tutorial here:
+This repository contains all of the code for interfacing with a 16x2 Character I2C LCD Display. This accompanies my YouTube tutorial here: https://www.youtube.com/watch?v=fR5XhHYzUK0
```
Comment (didaquis) [inline README.md]: Added link to video tutorial

**Comment by the-raspberry-pi-guy** [comment]
Awesome - just merged

**Pull request merged**